openfasoc/generators/gdsfactory-gen/pygen/pdk/sky130_mapped/sky130_add_npc.py

```diff
@@ -0,0 +1,100 @@
+from gdsfactory.component import Component
+from gdsfactory.polygon import Polygon
+from gdsfactory.geometry.boolean import boolean
+
+
+# slightly slower version
+#def sky130_add_npc(comp: Component) -> Component:
```

Comment:
There are some lines of commented code here. Is this code required? If not, it would be best to remove these lines to prevent confusion. If this is an alternative implementation, perhaps they can be combined and an argument can be used to switch between them?

Reply:
yes we can remove